app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx

```diff
@@ -163,6 +170,7 @@ const LineChart: FC<LineChartProps> = ({
 
   const options: uPlotOptions = {
     ...defaultOptions,
+    tzDate: ts => uPlot.tzDate(new Date(ts * 1e3), timezone),
```

Comment:
Let's use dayjs everywhere

Reply:
Why do you think dayjs is better to use in this particular place?

Reply:
I think it's better to have one approach to converting timezones everywhere. In our case, this is dayjs